### ISSUE TITLE
User leave Server

### ISSUE THREAD

**Issue by iHDxP** [Sank6/Discord-Bot-List#583]
Hello , i have suggestion to add message when user leave the server 

Ex : {bot.id} has been deleted as {user}  has left the server.`

**Comment by iHDxP** [comment]
https://github.com/Sank6/Discord-Bot-List/blob/master/src/bot/events/guildMemberRemove.js

` this.channels.cache.get(mod_log_id).send(`<@${bot.botid}> has been deleted as <@${member.user.id}> (${member.user.tag}) has left.`);`

Its not showing in mod_log_id specified channel

**Comment by SCSKasi** [comment]
@Sank6 any fix for this ?

**Comment by Robbie-Wittenhagen** [comment]
Do you have the channel id in the place of mod_log_id?

**Comment by iHDxP** [comment]
yes i do have mod_log_id placed in and i get all logs but when someone leave the server i dont get the message that he left ( in mod_log) , tested in fresh instalation still not working .
